### DISCUSSION TITLE
Change install path while doing silent install

### DISCUSSION THREAD

**Discussion by xxolaz** [coreybutler/nvm-windows#1017] [Support]
I was wondering if there is a solution to change the install path directory while silently installing, I have tried most parameters for other exe, but was wondering if there is specific ones for NVM or if there isn't yet. We are having a problem where when we install through Group Policy is going to C:\Windows\System32\config\systemprofile\AppData\Roaming\nvm and this can mess with UAC Permissions, when it also gets installed here the settings.txt file can't be loaded by NVM in cmd. Any information would be appreciated.

**Comment by coreybutler** [comment]
The primary installer is not setup to create silent installations with parameters. It may work, but I haven't tested it. We're addressing this in [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime), likely using a GPO-friendly MSI instead of an exe.

The current solution is to roll your own installer. The [manual installation guide](https://github.com/coreybutler/nvm-windows/wiki#manual-installation) walks through the setup, which is pretty simple as far as app installations go. Most organizations deploying this through AD are creating a package and setting the `NVM_HOME` environment variable to whatever directory they deploy to.

AD rollouts and generalized enterprise support are not a goal for NVM4W, but they are for Runtime... which we are now working on full-time. If you or anyone else reading this thread are interested in discussing enterprise challenges like this, I'm all ears while we roll out Runtime. I can provide access to a private channel for this type of conversation.
